### ISSUE TITLE
Feature request: warn vs error mode

### ISSUE THREAD

**Issue by backwardok** [chanzuckerberg/axe-storybook-testing#69]
For code bases that just start adding in a11y testing, there might be a mix of stories that are ready to be actively tested to prevent regressions and ones that aren't. Though stories could be turned off/skipped, it'd be nice to still be able to get metrics on what would fail with it enabled without failing the entire run. 

It'd be great if there were a config option where you could mark a story with a level, with options for `warn` or `error` (where `error` is the default). That way you could still run axe on everything that it could be run on, but you only fail if the ones that _shouldn't_ have errors started failing.

**Comment by ahuth** [comment]
Good point. Are you thinking instead of

```js
disabledRules: ['foo', 'bar', 'baz'],
```

it could be

```js
ruleOverrides: {
  foo: 'off',
  bar: 'warn',
  baz: 'warn',
},
```

or something like that?

If so, I like it 👍 . We can definitely add that or something similar.

**Comment by backwardok** [comment]
Yeah that definitely would be helpful! It would also be great to have a top level option similar to `skip` or instead of `skip` that could use all existing rules. 

Something like
```js
axe: {
  runStatus: 'off'/'warn'/'error'
}
```

Not tied to that wording at all, had a hard time thinking of a good name there 😂

**Comment by ahuth** [comment]
@backwardok I implemented this in #70 and merged it. Going to do some more testing, and probably publish a release with it tomorrow.

**Comment by ahuth** [comment]
Released in [v6.3.0](https://github.com/chanzuckerberg/axe-storybook-testing/releases/tag/v6.3.0) 🚀 

How the results are displayed leaves a little bit to be desired, but it's the simplest way I could do it without much deeper refactoring. Lemme know how it works for you!
